### PR TITLE
[FEATURE] Fix Bigquery/GCS integration tests

### PR DESCRIPTION
**Pull request by TrangPham**
Fixes a few of the GCS/BigQuery docs integration tests.

Ken and I were seeing these pass for us locally but fail in CI. It is now passing in CI.

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [x] Appropriate tests and docs have been updated

For more details, see our [Contribution Checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist), [Coding style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style), and [Documentation style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/docs_style).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Review by anthonyburdi — APPROVED** [review]
Nice find! Thanks for fixing this :bow:

File: tests/integration/test_definitions/bigquery/integration_tests.py [hunk 1]
```diff
@@ -108,20 +108,20 @@
...
-    #         (
-    #             "tests/integration/fixtures/split_and_sample_data/bigquery_connection_string.yml",
-    #             "connection_string.yml",
-    #         ),
-    #     ),
-    #     backend_dependencies=[BackendDependencies.BIGQUERY],
-    # ),
+    IntegrationTestFixture(
+        name="split_data_on_datetime_bigquery",
+        user_flow_script="tests/integration/db/test_sql_data_split_on_datetime_and_day_part.py",
```
Comment (Shinnnyshinshin) [inline tests/integration/test_definitions/bigquery/integration_tests.py:113]: 🚀 They work again!

File: tests/test_utils.py [hunk 2]
```diff
@@ -742,7 +742,14 @@ def load_data_into_test_bigquery_database_with_bigquery_client(
...
     )  # Make an API request.
-    job.result()  # Wait for the job to complete
+    load_job = job.result()  # Wait for the job to complete
+    if load_job.errors or load_job.error_result:
+        raise gx_exceptions.DatabaseConnectionError(
+            "Unable to load data into BigQuery. Load job errors: "
+            + str(load_job.errors)
+            + " Load job error result: "
+            + str(load_job.error_result)
+        )
```
Comment (Shinnnyshinshin) [inline tests/test_utils.py:752]: ❤️

**Review by Shinnnyshinshin — APPROVED** [review]
LGTM!

**Pull request merged**